### ISSUE TITLE
Update for V10?

### ISSUE THREAD

**Issue by BasiliskOnline** [FranzWarm/primalbox#1]
Hey this looks absolutely amazing! I was super stoked that it had a Foundry module, and it still looks great but functionality seems to have broken between the version 9 and 10 update! 
I'd love to see a version I could run in v10!

Edit: ahh looks like Sandbox system builder is also not updated for v10. Probably won't be able to update here until that happens at least eh? >.<

**Comment by FranzWarm** [comment]
Hey! Yes, unfortunately it seems that Sandbox is not yet ported to v10... So I guess the only way to use these sheets right now is to get back to Foundry v9. Sorry about that but there is much I can do about it at the moment!

**Comment by BasiliskOnline** [comment]
Yeah! I saw that, Luckily my foundry host on forge can just BARELY roll back to the foundry server i need. I hope they don't remove that when they update to 11 xD Thanks for the work anyway, the module looks real good :)

**Comment by BasiliskOnline** [comment]
Just wanted to stop in and mention that I noticed Sandbox has started updating again and is compatible with v11! IDK if you are in a place where you can or are interested in porting this in but I thought i'd mention it!

**Comment by csetom** [comment]
I do not know, how does sandbox works, but as I saw it, it need some CSS and position updates only. The mechanics worked, but in this shape, it is useless in v11.
